test/format/format_config.c

```diff
@@ -1449,6 +1451,88 @@ config_statistics(void)
     }
 }
 
+/*
+ * config_chunk_cache --
+ *     Chunk cache configuration.
+ */
+static void
+config_chunk_cache(void)
+{
+    char buf[128], chunkcache_storage_path[512];
+    const char *chunkcache_type;
+
+    chunkcache_type = NULL;
+
+    /* Chunkcache does not work unless tiered storage is configured. */
+    if (!g.tiered_storage_config) {
+        if (config_explicit(NULL, "chunk_cache") && GV(CHUNK_CACHE))
+            testutil_die(EINVAL,
+              "%s: chunkcache cannot be enabled unless tiered storage is configured.", progname);
+        return;
+    }
+
+    if (!config_explicit(NULL, "chunk_cache")) {
+        /*
+         * Make sure no configurations related to chunk caching are set if chunkcache is not
+         * enabled.
+         */
+        if (config_explicit(NULL, "chunk_cache.capacity") ||
+          config_explicit(NULL, "chunk_cache.chunk_size") ||
+          config_explicit(NULL, "chunk_cache.type") ||
+          config_explicit(NULL, "chunk_cache.storage_path"))
+            testutil_die(EINVAL,
+              "%s: Enable chunk caching (chunk_cache=on) to allow configuring other chunk cache "
+              "settings",
+              progname);
+
+        /* Enable chunkcache 50% of the time if not explicit set. */
+        testutil_snprintf(
+          buf, sizeof(buf), "chunk_cache=%s", mmrand(&g.data_rnd, 1, 100) <= 50 ? "on" : "off");
+        config_single(NULL, buf, false);
+    }
+
+    if (GV(CHUNK_CACHE)) {
+        if (config_explicit(NULL, "chunk_cache.type")) {
+            chunkcache_type = GVS(CHUNK_CACHE_TYPE);
+            if (strcmp(chunkcache_type, "FILE") != 0 && strcmp(chunkcache_type, "DRAM") != 0)
+                testutil_die(EINVAL, "illegal chunkcache.type configuration: %s", chunkcache_type);
+
+            if (GV(RUNS_IN_MEMORY) && strcmp(chunkcache_type, "FILE") == 0)
+                testutil_die(EINVAL,
+                  "%s: chunk caching cannot be enabled for in-memory runs as chunkcache.type is "
+                  "set to FILE.",
+                  progname);
+        } else {
+            if (GV(RUNS_IN_MEMORY))
+                config_single(NULL, "chunk_cache.type=DRAM", false);
+            else {
+                /*
+                 * Alternate between running chunk cache with the 'File' type and the 'DRAM' type.
+                 */
+                testutil_snprintf(buf, sizeof(buf), "chunk_cache.type=%s",
+                  mmrand(&g.data_rnd, 1, 100) <= 50 ? "DRAM" : "FILE");
+                config_single(NULL, buf, false);
+            }
+        }
+
+        if (strcmp(GVS(CHUNK_CACHE_TYPE), "FILE") == 0 &&
+          !config_explicit(NULL, "chunk_cache.storage_path")) {
+            testutil_snprintf(chunkcache_storage_path, sizeof(chunkcache_storage_path),
+              "chunk_cache.storage_path=%s/%s/WiredTigerChunkCache", getcwd(buf, sizeof(buf)),
+              g.home);
```

Comment:
I don't think this works if g.home is an absolute directory (e.g. `"/my/wthome"`).  Probably need to leave off the getcwd.

Reply:
Thanks for pointing this out. To make things simpler, I am planning to do the following - 

- If the storage path is not passed (i.e. by default), we create WTChunkCache in DB home directory.
- Otherwise, we check whether the path provided is absolute or relative and decide based on that.

Reply:
Updated in [bbd8d61](https://github.com/wiredtiger/wiredtiger/pull/9589/commits/bbd8d6164e258eb50c39cec4b8e62f5a280cc688)